.github/workflows/sycl-nightly.yml

```diff
@@ -332,20 +332,30 @@ jobs:
       toolchain_artifact_filename: ${{ needs.build-win.outputs.toolchain_artifact_filename }}
       sycl_cts_artifact: sycl_cts_bin_win
 
+  # Verification example:
+  # cosign-windows-amd64.exe verify-blob sycl_linux.tar.gz \
+  # --bundle sycl_linux.tar.gz.sigstore.json \
+  # --certificate-oidc-issuer https://token.actions.githubusercontent.com \
+  # --certificate-identity https://github.com/intel/llvm/.github/workflows/sycl-nightly.yml@refs/heads/sycl
   nightly_build_upload:
     name: Nightly Build Upload
     if: ${{ github.ref_name == 'sycl' }}
     needs: [ubuntu2204_build, build-win]
     runs-on: ubuntu-latest
     permissions:
       contents: write
+      id-token: write
```

Comment:
any idea why we need write for this?

Reply:
https://docs.sigstore.dev/quickstart/quickstart-ci/#signing-files-using-your-ci-system says:
`'id-token' needs write permission to retrieve the OIDC token, which is required for authentication.`

Reply:
comment makes no sense to me, retrieve seems like a read operation, but seems their problem not ours